### PR TITLE
Add QC notebook for post-processed ALFRESCO vegetation and flammability outputs

### PR DESCRIPTION
**Pull request by cstephen**
This notebook performs QC checks on the following post-processed ALFRESCO outputs:

- Vegetation mode
- Vegetation percent
- Flammability

These ALFRESCO outputs were generated using historical fire data from 1950 - 2021. We are currently using this notebook to perform QC checks on GeoTIFF deliverables for the SERDP Fish and Fire project, but it's also intended to be the first building block to getting these new ALFRESCO outputs incorporated into Rasdaman / ARDAC in the future.

The file system paths in the notebook should work as-is, and the notebook should be fairly self explanatory.

### CODE REVIEW

**Review by charparr — APPROVED** [review]
Hey this is the best QC notebook yet in my humble opinion!

I ran this notebook on Atlas and all cells executed successfully 💯 

I like the "guided discovery" style to the notebook and you joked about it being a cookbook, but it kind of is because it also is a very clear illustration of how to open, plot, and inspect the GeoTIFFs with Python. I'm really digging the 3x6 Vegetation Mode "small multiples" viz too.

Good idea on the "splotch correspondence" to show how the historical fire scars are indeed used in the model.

I'd stamp this dataset with the SNAP Platinum Seal of Excellence ™️

**Comment by charparr** [comment]
Oh I just noticed when looking at the notebook on GH that one of the cells was executed out-of-order so the cell with the code block
```python
for percent_sample_di in percent_sample_dis:
    for k in percent_sample_di.keys():
        assert(percent_sample_di[k]["min"] == 0.0)
        assert(percent_sample_di[k]["max"] == 100.0)
        assert(percent_sample_di[k]["mean"] < 70.0)
```
shows an error in the output.

Maybe just re-run all the cells in the notebook in order and then commit it so it looks clean when rendered on GH.

File: alfresco/pipeline/qc.ipynb [hunk 1]
```diff
@@ -0,0 +1,960 @@
...
+    "    print(\"Tags for \" + subpath + \":\")\n",
+    "    print(tags[subpath][0], end=\"\\n\\n\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "490e9c54",
+   "metadata": {},
+   "source": [
+    "Observe that the `TIFFTAG_IMAGEDESCRIPTION` and `TIFFTAG_SOFTWARE` tags are being used to communicate import information about these GeoTIFFs, including:\n",
```
Comment (kyleredilla) [inline alfresco/pipeline/qc.ipynb]:
```suggestion
    "Observe that the `TIFFTAG_IMAGEDESCRIPTION` and `TIFFTAG_SOFTWARE` tags are being used to communicate important information about these GeoTIFFs, including:\n",
```

**Review by charparr — APPROVED** [review]

**Review by kyleredilla — CHANGES_REQUESTED** [review]
This is an excellent example of a quality control check on a dataset!

The notebook executes as expected and is very easy to follow. The explanations / justifications are great. Agree with Charlie that this it the best one we've seen yet and should be paid attention to for future QC notebooks! Really thorough / clever tests.

I made a suggestion for a minor typo and also suggest re-executing as needed to ensure that all cells are executed prior to committing - looks like one of the "assert"-ing cells in the "Vegetation percent data checks" section was executed prematurely at one point and not re-run. After that, it's g2g.

**Review by kyleredilla — APPROVED** [review]

**Pull request merged**